src/scripts/search-engines/brave.ts

```diff
@@ -0,0 +1,87 @@
+import { SEARCH_ENGINES } from '../../common/search-engines';
+import { SearchEngine, SerpHandler } from '../types';
+import { handleSerp } from './helpers';
+
+function getSerpHandler(): SerpHandler {
+  return handleSerp({
+    globalStyle: {
+      '[data-ub-blocked="visible"]': {
+        backgroundColor: 'var(--ub-block-color, rgba(255, 192, 192, 0.5)) !important',
+      },
+      '.ub-button': {
+        color: 'var(--ub-link-color, var(--search-interactive-01))',
+      },
+      '.ub-button:hover': {
+        textDecoration: 'underline',
+      },
+      'div[id^=img-]': {
+        marginBottom: '80px !important',
+      },
+      'div[id^=img-][data-ub-blocked="visible"]': {
+        marginRight: '80px !important',
+      },
+      '[data-ub-blocked="visible"] > .img-url': {
+        backgroundColor: 'var(--ub-block-color, rgba(255, 192, 192, 0.5)) !important',
+      },
+      '.center-horizontally': {
+        flexWrap: 'wrap',
+      },
+    },
+    controlHandlers: [
+      {
+        target: '#filters-bar',
+        style: {
+          fontSize: '.66rem',
+          whiteSpace: 'nowrap',
+          padding: '4px',
+        },
+      },
+    ],
+    entryHandlers: [
+      // Web
+      {
+        target: '#results.section > .fdb',
```

Comment:
Wouldn't `.snippet` be better than `.fdb`? In your screenshot, the third and fourth results, which classes are `snippet` (not `snippet fdb`), are not filtered.

Reply:
@iorate Indeed you are right, sorry, I missed that.